flow/flow.go

```diff
@@ -1090,3 +1323,9 @@ func checkSystem() {
 		}
 	}
 }
+
+func check(e error) {
```

Comment:
Errors should be with unique diagnostic and use our instruments. Maybe you can combine errors in some functions together, because it is all about the same files. Remove this function.